.gitignore

```diff
@@ -117,7 +117,7 @@ celerybeat.pid
 *.sage.py
 
 # Environments
-.env
+.env/.env
```

Comment:
We should leave it as `.env` to prevent exposing the `credentials.json` here https://github.com/CodeForAfrica/gmmp/blob/master/gmmp/settings.py#L299

Reply:
I made the change so that we can include the `.env.template` which will act as a guide for what the data that the `.env` file needs. @kilemensi cc @esirK